client/network/src/service.rs

```diff
@@ -379,7 +379,8 @@ where
 				)
 				.substream_upgrade_protocol_override(upgrade::Version::V1Lazy)
 				.notify_handler_buffer_size(NonZeroUsize::new(32).expect("32 != 0; qed"))
-				.connection_event_buffer_size(1024);
+				.connection_event_buffer_size(1024)
+				.max_negotiating_inbound_streams(512);
```

Comment:
```suggestion
				.max_negotiating_inbound_streams(2048);
```

I shouldn't have to go in that PR to ask for that change